docs/README.rst

```diff
@@ -157,7 +219,41 @@ If you still feel confused then you can view more possible roles for our documen
 
 For more information, see: `Cross-referencing syntax <https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html>`_ in Sphinx documentation
 
+Docs troubleshooting
+====================
+
+``example_dags`` Apache license
+-------------------------------
+
+If you are creating ``example_dags`` directory, you need to create ``example_dags/__init__.py`` with Apache
+license or copy another ``__init__.py`` file that contains the necessary license.
```

Comment:
You can add empty `__init__.py` and pre-commit should add the licence automatically as well.